ckanext/bcgov/templates/package/snippets/map_view.html

```diff
@@ -22,8 +22,9 @@
       {% else %}
       	{% set map_service_url = pkg.preview_map_service_url %}
         	{% if map_service_url %}
-      		{% set wmsservice_url = map_service_url + "&wmslayers=" + pkg.layer_name + "&ll=" + pkg.preview_latitude + "," + pkg.preview_longitude + "&z=" + pkg.preview_zoom_level %}
-      		{% set resource_url = "//apps.gov.bc.ca/pub/dmf-viewer/?siteid=7535188336326689232&maponly&wmsservices=" + wmsservice_url %}
+          {% set wmsservice_url = map_service_url + "&wmslayers=" + pkg.layer_name + "&ll=" + pkg.preview_latitude + "," + pkg.preview_longitude + "&z=" + pkg.preview_zoom_level %}
+          {% set wmsservice_url = pkg.layer_name + "&smk-show-tool=zoom|pan|baseMaps&smk-center=" + pkg.preview_longitude + "," + pkg.preview_latitude + "," + pkg.preview_zoom_level %}
```

Comment:
Should longitude precede latitude?  It looks like the required pattern may be latitude first, then longitude.

Reply:
The way I tested it worked with dev data

Reply:
OK, then let's give it a try.

Reply:
Reviewed this again.  Previous order was latitude, then longitude.

Reply:
for SMK as I recall its X,Y order so long lat is correct.

Reply:
Right I switched it with direction from @franTarkenton

Reply:
Thanks both for clarification.